docs/design.md

```diff
@@ -178,6 +215,12 @@ The process is as follows.
    - If the quorum of the cluster does not have data, users need to recover the cluster from a backup.
 4. The operator initializes the new blank Pods as new slaves.
 
+### How to collect metrics for Prometheus
+
+To avoid unncessary restart when the operator is updated, we prepare external Pods which collect prometheus metrics over network.
```

Comment:
```suggestion
To avoid unnecessary restart when the operator is upgraded, we prepare external Pods which collect mysqld metrics over the network, and export them as Prometheus metrics.
```